### PR TITLE
{math}[GCC/10.2.0,iccifort/2020.4.304] SLATEC v4.1

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/SLATEC/SLATEC-4.1-slatec4linux-fPIC.patch [hunk 1]
```diff
@@ -0,0 +1,9 @@
+diff -Nru slatec4linux-orig/static/makefile slatec4linux/static/makefile
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/SLATEC/SLATEC-4.1-slatec4linux-fPIC.patch:1]: please add description/reason and author

Comment (SimonPinches) [reply easybuild/easyconfigs/s/SLATEC/SLATEC-4.1-slatec4linux-fPIC.patch:3]: @migueldiascosta - done!  (After a rather lengthy delay - sorry!)

Comment (migueldiascosta) [reply easybuild/easyconfigs/s/SLATEC/SLATEC-4.1-slatec4linux-fPIC.patch:3]: @SimonPinches I'm afraid you'll also need to change the patch checksum :)

**Comment by boegel** [comment]
closing this since `GCC/10.2.0` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Comment by hattom** [comment]
FYI, for anyone who stumbles across this; it seems like upstream SLATEC might have been broken late last year.

**Pull request closed without merging**